src/pages/workspace/reimburse/WorkspaceReimburseView.js

```diff
@@ -188,31 +191,52 @@ class WorkspaceReimburseNoVBAView extends React.Component {
                     </View>
                 </Section>
 
-                <Section
-                    title={this.props.translate('workspace.reimburse.unlockNextDayReimbursements')}
-                    icon={Illustrations.JewelBoxGreen}
-                >
-                    <View style={[styles.mv4]}>
-                        <Text>{this.props.translate('workspace.reimburse.unlockNoVBACopy')}</Text>
-                    </View>
-                    <Button
-                        text={this.props.translate('workspace.common.bankAccount')}
-                        onPress={() => Navigation.navigate(ROUTES.getWorkspaceBankAccountRoute(this.props.policyID))}
-                        icon={Expensicons.Bank}
-                        style={[styles.mt4]}
-                        iconStyles={[styles.mr5]}
-                        shouldShowRightIcon
-                        extraLarge
-                        success
-                    />
-                </Section>
+                {Boolean(!this.props.hasVBA) && (
```

Comment:
`!this.props.hasVBA` is already boolean.

Reply:
doesn't `this.props.hasVBA` also is already boolean?
We get it from:
https://github.com/Expensify/App/blob/9ef8ac261667371a4818f55f7ee883abeb31312a/src/pages/workspace/WorkspacePageWithSections.js#L74
If so, I think we won't need to cast it to boolean in:
```js
{Boolean(this.props.hasVBA) && (
```